src/protocol/mul/malicious.rs

```diff
@@ -83,11 +84,19 @@ impl<'a, F: Field> MaliciouslySecureMul<'a, F> {
         // being clever and assuming a clean context...
         let duplicate_multiply_ctx = self.ctx.narrow(&Step::DuplicateMultiply);
         let random_constant_prss = self.ctx.narrow(&Step::RandomnessForValidation).prss();
-        let (ab, rab) = try_join(
-            SecureMul::new(self.ctx, self.record_id).execute(a.x(), b.x()),
-            SecureMul::new(duplicate_multiply_ctx, self.record_id).execute(a.rx(), b.x()),
-        )
-        .await?;
+        let (ab, rab) = {
+            // Convince compiler that neither a nor b will be used across the await point
+            // to relax the requirement for either of them to be Sync
```

Comment:
We call this function from `SecureMul` trait implementation. The trait definition uses `async_trait` crate because Rust does not support async functions inside traits (yet, 1.65 is going to change it). `async_trait` requires return to be `Send` (we can opt out but I don't think we should): https://docs.rs/async-trait/latest/async_trait/#explanation

future returned by `MaliciouslySecureMul` wasn't `Send` and this change makes it `Send`.